libs/checkpoint-mongodb/langgraph/checkpoint/mongodb/aio.py

```diff
@@ -0,0 +1,306 @@
+from contextlib import asynccontextmanager
+from typing import (
+    Any,
+    AsyncIterator,
+    Dict,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+)
+
+from langchain_core.runnables import RunnableConfig
+from pymongo import AsyncMongoClient, UpdateOne
+from pymongo.asynchronous.database import AsyncDatabase
```

Comment:
We should be using Motor until we take Async PyMongo out of beta

Reply:
I switched to Motor.